packages/block-editor/src/components/rich-text/index.native.js

```diff
@@ -660,7 +660,7 @@ function RichTextWrapper(
 					{ nestedIsSelected && hasFormats && (
 						<FormatToolbarContainer
 							inline={ inlineToolbar }
-							anchorRef={ fallbackRef.current }
+							editableContentRef={ fallbackRef }
```

Comment:
As explained before, the `FormatToolbarContainer` expects `editableContentRef` to be a React ref object

Reply:
> Doesn't this FormatToolbarContainer refer to `format-toolbar-container.native.js` though?

🤦  Not sure how I missed that file. I will revert changes to this file.

> I'm confused because that native component doesn't take any props.

That is interesting.

@dcalhoun @fluiddot @geriux , feel free to ignore my previous request to review this PR. You may still want to investigate separately why `FormatToolbarContainer` is receiving the `inline` and `anchorRef` props, despite the native component not accepting any.

Reply:
It appears the unused props passed to `FormatToolbarContainer` result from [splitting of once shared logic](https://github.com/WordPress/gutenberg/commit/b4f0d82aa2f19d5205f83ceccffde623350aae8c#diff-1bf4290b2ff8a9f2d347d3450576c98613f26f26f4ce05b850cf232483f7b28cR611-R614) without removing unused, web-specific props. It is a complex component, so it is not surprising this could occur. Likely worthy of future clean up. Thank you for calling it to our attention. 🙇🏻